rabbitmq-lambda-tf/example-pattern.json

```diff
@@ -0,0 +1,60 @@
+{
+  "title": "RabbitMQ to AWS Lambda",
+  "description": "Deploys a Lambda function, a RabbitMQ queue and the IAM permissions required to run the application. RabbitMQ invokes the Lambda function when new messages are available.",
+  "language": "YAML",
+  "level": "200",
+  "framework": "Terraform",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This template deploys a Lambda function, a RabbitMQ queue and the IAM permissions required to run the application.",
+      "RabbitMQ invokes the Lambda function when new messages are available."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/rabbitmq-lambda-tf",
+      "templateURL": "serverless-patterns/rabbitmq-lambda-tf",
+      "projectFolder": "rabbitmq-lambda-tf",
+      "templateFile": "main.tf"
+    }
+  },
+  "resources": {
+    "bullets": [
+      {
+        "text": "Using Amazon MQ for RabbitMQ as an event source for Lambda",
+        "link": "https://aws.amazon.com/blogs/compute/using-amazon-mq-for-rabbitmq-as-an-event-source-for-lambda/"
+      },
+      {
+        "text": "RabbitMQ",
+        "link": "https://www.rabbitmq.com/"
+      }
+    ]
+  },
+	"deploy": {
+		"text": [
+		          "terraform init",
+		          "terraform apply"
+		        ]
+	},
+	"testing": {
+		"text": [
+		"See the Github repo for detailed testing instructions."
+		]
+	},
+	"cleanup": {
+		"text": [
+		          "terraform destroy",
+		          "terraform show"
```

Comment:
Is that really necessary?

Reply:
Yes, it is needed to confirm there are no resources exist after issuing the command terraform destroy.